### PR TITLE
Issue78 solution

### PR DESCRIPTION
**Pull request by MKletz**
- https://github.com/techservicesillinois/SecOps-Powershell-CISDSC/issues/78

- Added support for tracking validation blocks for resource parameters. These have to be statically documented and not generated since these require looking into documentation to see what the actual setting supports vs what the data type min/max is.

### CODE REVIEW

**Comment by edthedev** [comment]
Can this new functionality be unit tested? 

The wording in the ticket set off my "there may now be two ways to do this, and only one may get regularly tested" alert.

**Comment by MKletz** [comment]
> Can this new functionality be unit tested?
> 
> The wording in the ticket set off my "there may now be two ways to do this, and only one may get regularly tested" alert.

yes! over sight on my part. Shouldn't be too difficult.

**Comment by MKletz** [comment]
@edthedev tests added

File: docs/new_resources.md [hunk 1]
```diff
@@ -52,7 +53,12 @@ Successfully generated resources will be placed into the generated composite res
 
 4) Resolve errors identified in the previous step. Potential causes and resolutions are outlined in the [static corrections](./static_corrections.md) documentation.
 
-5) Verify your results with the [CIS-CAT](https://workbench.cisecurity.org/files/2151) assessment tool available from the [CIS workbench](https://workbench.cisecurity.org/). A HTML report from the tool will be required later. Documentation for the tool can be found [here](https://ccpa-docs.readthedocs.io/en/latest/User%20Guide%20for%20CLI/).
+5) Verify the appropriate validation for generated parameters via MS documentation or otherwise and add it to the [parameter validation](/parameter_validations.csv) documentation.
+
+6) Regenerate your resource after completing the previous two steps.
+
+7) Verify your results with the [CIS-CAT](https://workbench.cisecurity.org/files/2151) assessment tool available from the [CIS workbench](https://workbench.cisecurity.org/). A HTML report from the tool will be required later. Documentation for the tool can be found [here](https://ccpa-docs.readthedocs.io/en/latest/User%20Guide%20for%20CLI/).
```
Comment (edthedev) [inline docs/new_resources.md:60]: Can we answer here - required by who (or by what tool)?

**Review by edthedev — APPROVED** [review]

File: docs/new_resources.md (hunk 1 above)
Comment (edthedev) [reply docs/new_resources.md:60]: No need to hold up merging over this suggestion - just could be helpful to add if it's an easy answer.

Comment (MKletz) [reply docs/new_resources.md]: Clarified

**Review by baristaTam — APPROVED** [review]

**Review by edthedev — APPROVED** [review]

**Pull request merged**